### PR TITLE
[Tests] Test loggers video saving

### PR DESCRIPTION
**Pull request by bashnick**
https://github.com/facebookresearch/rl/issues/396

## Description

Added the tests for the log_video methods in Tensorboard, CSVLogger and WandbLogger.

## Motivation and Context

Fixes an open issue, #396 

## Types of changes

What types of changes does your code introduce? Remove all that do not apply:

- [x] New feature (non-breaking change which adds core functionality)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/facebookresearch/rl/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: test/test_loggers.py [hunk 1]
```diff
@@ -42,6 +42,48 @@ def test_log_scalar(self, steps):
...
+        torch.manual_seed(0)
+        with tempfile.TemporaryDirectory() as log_dir:
+            exp_name = "ramala"
+            logger = TensorboardLogger(log_dir=log_dir, exp_name=exp_name)
+
+            # creating a sample video (T, C, H, W), where T - number of frames,
+            # C - number of image channels (e.g. 3 for RGB), H, W - image dimensions.
+            # the first 64 frames are black and the next 64 are white
+            video = torch.cat(
+                (torch.zeros(64, 1, 32, 32), torch.mul(torch.ones(64, 1, 32, 32), 255))
```
Comment (vmoens) [inline test/test_loggers.py:56]: you can use `torch.full` here for a tensor full of 255

**Review by vmoens — COMMENTED** [review]
LGTM, do you want to test that the video looks like what it should? Also any idea of how to test the fps for tensorboard?

File: test/test_loggers.py (hunk 1 above)
Comment (bashnick) [reply test/test_loggers.py:56]: yes, thanks!

Comment (bashnick) [reply test/test_loggers.py:56]: Re the "fps for tensorboard", the problem is that the return format for the video is an encoded_string, which has the same length for any video. I didn't find a nice method to convert it back to tensor or video object.
Re the "want to test that the video looks like what it should": 
- I am testing it in the CSVLogger
- For the WandbLogger and Tensorboard haven't found a nice method to do that.

Comment (vmoens) [reply test/test_loggers.py]: Ok I think we're good to go as it is!

**Review by vmoens — APPROVED** [review]
LGTM thanks for this

**Pull request merged**